packages/react/src/lib/accessibility/Behaviors/Accordion/accordionTitleBehavior.ts

```diff
@@ -0,0 +1,31 @@
+import { Accessibility } from '../../types'
+import * as keyboardKey from 'keyboard-key'
+
+/**
+ * @specification
+ */
+const accordionTitleBehavior: Accessibility = (props: any) => {
+  const isHeading = /(h\d{1})$/.test(props.as)
+  return {
+    attributes: {
+      root: {
+        role: isHeading ? undefined : 'heading',
+        'aria-level': isHeading ? undefined : 3,
+      },
+      button: {
+        'aria-expanded': !!props.active,
+        'aria-disabled': !!(props.active && props.cannotBeClosed),
+        'aria-owns': props.contentId,
```

Comment:
I see from spec and example they suggest using `aria-controls`, not `aria-owns` or I might missing something
`The accordion header button element has aria-controls set to the ID of the element containing the accordion panel content.`

Reply:
no, you are right.